packages/apps/crm/src/components/organisms/DropdownOpportunityView/DropdownOpportunityView.tsx

```diff
@@ -0,0 +1,92 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2023 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useMemo} from 'react';
+import {StyleSheet, View} from 'react-native';
+import {Text} from '@axelor/aos-mobile-ui';
+import {isEmpty, useNavigation, useTranslator} from '@axelor/aos-mobile-core';
+import {OpportunityCard} from '../../molecules';
+import {getLastItem} from '../../../utils/list';
+
+interface Opportunity {
+  id: number;
+  amount: string;
+  expectedCloseDate: string;
+  name: string;
+  opportunityRating: number;
+  opportunitySeq: string;
+  allOpportunityStatus?: any;
+  currencySymbol?: string;
+  opportunityStatus?: any;
+}
+
+interface DropdownOpportunityViewProps {
+  opportunityList: Opportunity[];
+  opportunityStatusList?: any;
+}
+
+const DropdownOpportunityView = ({
+  opportunityList,
+  opportunityStatusList,
+}: DropdownOpportunityViewProps) => {
+  const I18n = useTranslator();
+  const navigation = useNavigation();
+
+  const lastOpportunity = useMemo(() => {
+    return getLastItem(opportunityList);
```

Comment:
This won't work. Opportunity does not have a startDateTime props. Please pay attention to object domain.
last opportunity should be calculated with expectedCloseDate.
More over, even if Opportunity has a startDateTime it won't have worked because you didn't add the field in the fields register.